integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -4137,6 +4141,39 @@ msgstr "Nicht geprüfte Links"
 msgid "Total links"
 msgstr "Links insgesamt"
 
+#: cms/templates/analytics/_page_hix_widget.html
+msgid "Page HIX list"
+msgstr "Seitenliste mit HIX-Werten"
+
+#: cms/templates/analytics/_page_hix_widget.html
+msgid "The list below shows pages ascendingly ordered by HIX value."
+msgstr "Die Liste zeigt Seiten nach HIX-Wert aufsteigend."
+
+#: cms/templates/analytics/_page_hix_widget.html
+msgid " of all pages are ready for automatic translation."
+msgstr " der Seiten sind für maschienelle Übersetzng bereit."
+
+#: cms/templates/analytics/_page_hix_widget.html
+msgid "HIX score is too low for automatic translation if marked red."
+msgstr ""
+"HIX-Wert ist für maschienelle Übersetzung zu schlecht wenn rot markiert."
```

Comment:
```suggestion
"Um Seiten maschinell übersetzen zu können, muss die Seite einen HIX-Wert von mindestens 15 haben. Wenn die Zeile rot markiert ist, ist der HIX-Wert der entsprechenden Seite nicht ausreichend."
```

I think it would be helpful for the CMS user to know how high the HIX value has to be in order to be able to use MT for that page

Reply:
Here as well. The German translation should actually be a translation of the English source text and not add additional information that's hidden when the user uses the English UI.